app/models/group.rb

```diff
@@ -1,5 +1,11 @@
 class Group < ActiveRecord::Base
   has_many :users
 
-  validates :name, :address, presence: true, uniqueness: true
+  validates :street, :city, presence: true
+  validates :number, :zip_code,  numericality: { only_integer: true }
```

Comment:
Are there any benefits of having the zipcode be an integer as opposed to a string?